velox/dwio/parquet/writer/arrow/util/Hashing.h

```diff
@@ -252,14 +254,15 @@ class HashTable {
   };
 
   HashTable(MemoryPool* pool, uint64_t capacity) : entries_builder_(pool) {
-    DCHECK_NE(pool, nullptr);
+    VELOX_DCHECK_NOT_NULL(pool);
     // Minimum of 32 elements
     capacity = std::max<uint64_t>(capacity, 32UL);
     capacity_ = ::arrow::bit_util::NextPower2(capacity);
     capacity_mask_ = capacity_ - 1;
     size_ = 0;
 
-    DCHECK_OK(UpsizeBuffer(capacity_));
+    auto status = UpsizeBuffer(capacity_);
+    VELOX_DCHECK(status.ok(), status.ToString());
```

Comment:
We may want to define `VELOX_DCHECK_OK()` that is a specialized `DCHECK` for `arrow::Status`.